lib/ui/views/app_selector/app_selector_viewmodel.dart

```diff
@@ -69,6 +70,36 @@ class AppSelectorViewModel extends BaseViewModel {
     return true;
   }
 
+  Future<void> searchSuggestedVersionOnWeb(BuildContext context,
+      {required String packageName, required String appName, bool isInstalled = false,}) async {
```

Comment:
appName doesn't exist for apps that aren't installed, what's the usage of it here? it's also less reliant, package name tends to give better results